build/__snapshots__/features-meta.json

```diff
@@ -748,6 +748,12 @@
 		"description": "Retries downloading images that failed downloading due to GitHub limited proxying.",
 		"screenshot": "https://user-images.githubusercontent.com/14858959/64068746-21991100-cc45-11e9-844e-827f5ac9b51e.png"
 	},
+	{
+		"id": "render-conventional-commit-types",
```

Comment:
The closest feature we have is called `parse-backticks` so we can name this `parse-conventional-commits` or honestly I just want `conventional-commits`. I don’t expect additional related features due to what @134130 said so we can keep it simple with two words.

Reply:
I guess `conventional-commits` would work, yeah